src/main/java/io/github/swagger2markup/internal/component/PathOperationComponent.java

```diff
@@ -73,28 +75,53 @@ public PathOperationComponent(Swagger2MarkupConverter.Context context,
         this.responseComponent = new ResponseComponent(context, definitionDocumentResolver);
     }
 
-    public static PathOperationComponent.Parameters parameters(PathOperation operation) {
-        return new PathOperationComponent.Parameters(operation);
+    public static PathOperationComponent.Parameters parameters(PathOperation operation, List<PageBreakLocations> pageBreakLocations) {
```

Comment:
You don't have to add pageBreakLocations as a parameter, because all components extend `MarkupComponent` which has access to the `Swagger2MarkupConfig`.